src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/DependencyInjection/AddHttpClientDefaultsTest.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net.Http;
+using Microsoft.Extensions.Http;
+using Xunit;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    // These are mostly integration tests that verify the configuration experience.
+    public class AddHttpClientDefaultsTest
+    {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        public void AddHttpClientDefaults_WithNameConfig_NameConfigUsed()
+        {
+            // Arrange
+            var serviceCollection = new ServiceCollection();
+
+            // Act1
+            serviceCollection.AddHttpClient("example.com", c => c.BaseAddress = new Uri("http://example.com/"));
+            serviceCollection.AddHttpClientDefaults().ConfigureHttpClient(c => c.BaseAddress = new Uri("http://default.com/"));
+
+            var services = serviceCollection.BuildServiceProvider();
```

Comment:
using var?

Reply:
Yup

Reply:
I meant to dispose built service collection 🙂
`using var services = ...`

Reply:
Ohhhh.

No, that's not necessary. The test isn't checking that disposable things are disposed of. It wouldn't hurt, but none of the other tests dispose the service provider.